src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Alloying.java

```diff
@@ -89,11 +99,13 @@ public boolean removeByInputsAndOutput(FluidStack output, FluidStack... inputs)
         return false;
     }
 
+    @MethodDescription(description = "groovyscript.wiki.removeAll")
```

Comment:
if you do this instead, the method will be documented in the examples file (commented out). the priority is used to make it be the last entry, since having it in the middle doesnt make much sense 
```suggestion
    @MethodDescription(description = "groovyscript.wiki.removeAll", priority = 2000, example = @Example(commented = true))
```